bo4e/models/bo/Dokument.json

```
@@ -2,20 +2,20 @@
   "additionalProperties": true,
   "description": "A generic document reference like for bills, order confirmations and cancellations",
   "properties": {
-    "versionstruktur": {
+    "version": {
       "default": "2",
-      "title": "Versionstruktur",
+      "title": "Version",
```

```suggestion
      "title": " Version",
```
Rein aus Konsistenz.